### ISSUE TITLE
API to support secure requests

### ISSUE THREAD

**Issue by olgamaker** [Hipo/university-domains-list#256]
Hi,

I'm wondering if there's a plan for the hosted API to support requests via https?

Thanks

**Comment by yigitguler** [comment]
Yes, we can add support for that. Could you please open an issue on https://github.com/Hipo/university-domains-list-api?

**Comment by ceydaduzgec** [comment]
Hello @olgamaker, thanks for your question. But I am closing this issue since it already has a related issue in the [ https://github.com/Hipo/university-domains-list-api/issues/19]( https://github.com/Hipo/university-domains-list-api/issues/19) repo.
